WordPress/Classes/ViewRelated/Menus/MenuDetailsViewController.m

```diff
@@ -79,7 +79,6 @@ - (void)setupTrashButton
     [trashButton setImage:[UIImage gridiconOfType:GridiconTypeTrash] forState:UIControlStateNormal];
     [trashButton addTarget:self action:@selector(trashButtonPressed) forControlEvents:UIControlEventTouchUpInside];
     trashButton.backgroundColor = [UIColor clearColor];
-    trashButton.adjustsImageWhenHighlighted = YES;
```

Comment:
This is no-op because `true` is the default value. (Yes, it's deprecated in iOS 15).